services/director-v2/tests/unit/test_db_comp_tasks.py

```diff
@@ -49,12 +49,18 @@ def all_frontend_services(
 def test_only_filepicker_service_gets_some_service_details(
     all_frontend_services: Set[str],
 ):
-    for frontend_service in all_frontend_services:
-        service_key_version = ServiceKeyVersion(key=frontend_service, version="24.34.5")
-        # check that it does not assert
-        service_docker_data: ServiceDockerData = _get_fake_service_details(
-            service_key_version
-        )
-        if service_docker_data:
-            assert service_docker_data.key == "simcore/services/frontend/file-picker"
-            assert service_docker_data.outputs["outFile"].property_type == "data:*/*"
+    # FIXME: PC->SAN: this is ONLY for filepicker??
```

Comment:
@sanderegg based on the test name, is this really ONLY for the file-picker or can we use for all?